admin-client/src/components/Nav.svelte

```diff
@@ -89,6 +89,13 @@
             </a>
           {/if}
         </li>
+        <li class="usa-nav__primary-item">
+          {#if authenticated}
```

Comment:
Should we move this out to cover all of the authenticated links?

Reply:
yeah, looks like they are stacked from the right so it looks ok!

Reply:
done